src/stim/mem/simd_bit_table.inl

```diff
@@ -90,12 +90,28 @@ simd_bit_table<W> simd_bit_table<W>::inverse_assuming_lower_triangular(size_t n)
     return result;
 }
 
+/// SIMD transpose:
+/// Suppose we have a matrix with minor axis contiguous in memory
+///
+/// A00  A01  ...  A0n
+/// A10  A11  ...  A1n
+///           ...
+/// An0  An1  ...  A2n
```

Comment:
Why is the maximum index `n` for both axes? Normally the major axis has W times fewer possible indices due to the bit packing of the minor axis.

Reply:
have removed this now that I understand the conventions being used better